### ISSUE TITLE
UnmappedValueException on client login.

### ISSUE THREAD

**Issue by alvileg** [GeyserMC/MCProtocolLib#691]
The following error is being produced when the client logs in, or when the player gets opped. 
It seems that this only happens when the commands are being [declared](https://wiki.vg/Protocol#Declare_Commands).

Here is the Exception:
```
Internal Exception: com.github.steveice10.mc.protocol.data.UnmappedValueException: Value minecraft:resource_or_tag has no mapping for key class com.github.steveice10.mc.protocol.data.game.command.CommandParser.
```

Paper Version: `1.18.2`
MCProtocolLib Version: `latest: 1.18-3`

**Comment by alvileg** [comment]
Investigating further, the value is indeed [Mapped](https://github.com/GeyserMC/MCProtocolLib/blob/master/src/main/java/com/github/steveice10/mc/protocol/data/game/command/CommandParser.java#L51), added in the [Merge branch 'feature/1.18.2'](https://github.com/GeyserMC/MCProtocolLib/commit/7efa6360dc3d54ddac9f52fb9ea090277f827328), but it was not included in the release [1.18-3](https://github.com/GeyserMC/MCProtocolLib/releases/tag/1.18-3)

Would it be possible to make a new release with the latest changes?

Thanks

**Comment by alvileg** [comment]
This was fixed by the latest [1.18.2-1 Release](https://github.com/GeyserMC/MCProtocolLib/releases/tag/1.18.2-1)

Thanks.
